.github/workflows/docs-deployment.yml

```
@@ -0,0 +1,50 @@
+name: Docs Deployment
+
+on:
+  push:
+    branches:
+      - master
```

It's better to deploy automatically when we cut a new release. If we need to update the docs without releasing a version we can trigger the workflow manually.

I want to prevent an out-synced doc if we merge a PR today and release it in a week.

```suggestion
  release:
    types: [published]
```

---

This has been updated.